internal/cli/start.go

```diff
@@ -93,210 +90,10 @@ For all the example cases, see: https://docs.kairos.io .
 				},
 			},
-			{
-				Name:      "register",
-				UsageText: "register --reboot --device /dev/sda /image/snapshot.png",
-				Usage:     "Registers and bootstraps a node",
-				Description: `
-		Bootstraps a node which is started in pairing mode. It can send over a configuration file used to install the kairos node.
-		
-		For example:
-		$ kairos register --config config.yaml --device /dev/sda ~/Downloads/screenshot.png
-		
-		will decode the QR code from ~/Downloads/screenshot.png and bootstrap the node remotely.
-		
-		If the image is omitted, a screenshot will be taken and used to decode the QR code.
-		
-		See also https://kairos.io/docs/getting-started/ for documentation.
-		`,
-				ArgsUsage: "Register optionally accepts an image. If nothing is passed will take a screenshot of the screen and try to decode the QR code",
-				Flags: []cli.Flag{
-					&cli.StringFlag{
-						Name:     "config",
-						Usage:    "Kairos YAML configuration file",
-						Required: true,
-					},
-					&cli.StringFlag{
-						Name:  "device",
-						Usage: "Device used for the installation target",
-					},
-					&cli.BoolFlag{
-						Name:  "reboot",
-						Usage: "Reboot node after installation",
-					},
-					&cli.BoolFlag{
-						Name:  "poweroff",
-						Usage: "Shutdown node after installation",
-					},
-					&cli.StringFlag{
-						Name:  "log-level",
-						Usage: "Set log level",
-					},
-				},
-
-				Action: func(c *cli.Context) error {
-					var ref string
-					if c.Args().Len() == 1 {
-						ref = c.Args().First()
-					}
-
-					return register(c.String("log-level"), ref, c.String("config"), c.String("device"), c.Bool("reboot"), c.Bool("poweroff"))
-				},
-			},
-			{
-				Name:      "bridge",
-				UsageText: "bridge --network-token XXX",
-				Usage:     "Connect to a kairos VPN network",
-				Description: `
-		Starts a bridge with a kairos network or a node. 
-		
-		# With a network
-		
-		By default, "bridge" will create a VPN network connection to the node with the token supplied, thus it requires elevated permissions in order to work.
-		
-		For example:
-		
-		$ sudo kairos bridge --network-token <TOKEN>
-		
-		Will start a VPN, which local ip is fixed to 10.1.0.254 (tweakable with --address).
-		
-		The API will also be accessible at http://127.0.0.1:8080
-		
-		# With a node
-		
-		"kairos bridge" can be used also to connect over to a node in recovery mode. When operating in this modality kairos bridge requires no specific permissions, indeed a tunnel
-		will be created locally to connect to the machine remotely.
-		
-		For example:
-		
-		$ kairos bridge --qr-code-image /path/to/image.png
-		
-		Will scan the QR code in the image and connect over. Further instructions on how to connect over will be printed out to the screen.
-		
-		See also: https://docs.kairos.io/after_install/troubleshooting/#connect-to-the-cluster-network and https://docs.kairos.io/after_install/recovery_mode/
```

Comment:
I couldn't find a reference to the first link in the new docs. If you know which one it is, pls let me know so I can update it